scalableminds/webknossos
# Time tracking notification

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- adds task time limits to projects
- after exceeding the limit when working on a task an admin is notified via email

### Steps to test:
- Frontend Changes:
  1. Create a Project and set a time limit
  2. Edit the project and time limit
  3. values should update accordingly
  4. (checkout task type view. There should be no more references to time limits)

### Issues:
- fixes #585

------
